### DISCUSSION TITLE
Thubnails Carousel Handle

### DISCUSSION THREAD

**Discussion by Naveen-sk** [igordanchenko/yet-another-react-lightbox#303] [Q&A] [answered]
Is that possible to add the previous and next icon button for thubnails carousel and make them work regarding the finite functionality says, if there is no next/previous image disable the button. Also I want the list thubnails from the start if the preload set as slides.lenght

Thanks in Advance....

**Comment by igordanchenko** [comment, marked as answer]
I'm afraid this is not supported in the current implementation.
